### PR TITLE
build lua2 backend packages

### PR DESCRIPTION
**Pull request by cmouse**
### Short description
Build packages for lua2 backend

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled and tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by cmouse** [comment]
I have no idea if this is going to work or not =)

File: build-scripts/build-auth-rpm [hunk 1]
```diff
@@ -340,6 +340,15 @@ Requires: %{name}%{?_isa} = %{version}-%{release}
 %description backend-lua
 This package contains the lua backend for %{name}
 
+%package backend-lua
```
Comment (pieterlexis) [inline build-scripts/build-auth-rpm]: `s/lua/lua2/`

File: build-scripts/build-auth-rpm [hunk 2]
```diff
@@ -340,6 +340,15 @@ Requires: %{name}%{?_isa} = %{version}-%{release}
...
 
+%package backend-lua2
+Summary: New Lua backend for %{name}
+Group: System Environment/Daemons
+Requires: %{name}%{?_isa} = %{version}-%{release}
+%global backends %{backends} lua2
+
+%description backend-lua2
+This package contains the new lua backend for %{name}
+
```
Comment (Habbie) [inline build-scripts/build-auth-rpm]: I'm pretty sure you also need to tell the RPM what files go into the package.

Comment (cmouse) [reply build-scripts/build-auth-rpm]: I hope I finally found all them places. =)

**Pull request merged**